specs-go/version.go

```diff
@@ -25,7 +25,7 @@ const (
 	VersionPatch = 0
 
 	// VersionDev indicates development branch. Releases will be empty string.
-	VersionDev = "-dev"
+	VersionDev = "-rc.2-dev"
```

Comment:
should be just `-rc2`:
```suggestion
	VersionDev = "-rc2"
```

Reply:
or `-rc.2` (no dev). Right?

Reply:
well, the idea is that `main` always stays with the `-dev` suffix, and we tag the commit in the middle

Reply:
right but why `"-rc.2-dev"` and not `"-rc.2"` ?

Reply:
going off of https://github.com/opencontainers/image-spec/pull/1049

Reply:
er, let me rephrase - why `"-rc.2-dev"` and not `"-dev"` ? (for the second commit)

Reply:
yeah ok fair. I can change it to just `-dev`